index.html

```diff
@@ -1,57 +1,158 @@
+          <li class="txt" id="role">Back End Dev</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point-2"></li>
+          <li class="txt" id="year">2015</li>
+        </ul>
+      </div>
+      <div>
+        <p id="description-2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="tags">
+        <ul id="tags-list">
+          <li id="h-tag"><a href="#" class="tag" id="html">HTML</a></li>
+          <li id="c-tag"><a href="#" class="tag" id="css">CSS</a></li>
+          <li id="j-tag"><a href="#" class="tag" id="JavaScript">JavaScript</a></li>
+        </ul>
+      </div>
+      <div class="actions">
+        <button disabled class="NormalB">See project</button>
+      </div>
+    </section>
+  </div>
+
+
 
+  <section class="About-My">
+    <div class="Top">
+      <section class="Mytxt">
+        <div>
+          <h1 class="Myself">About Myself</h1>
+          <p id="description">
+            Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+            me.
+          </p>
+          <h2 id="connect" class="connect-title">LET’S CONNECT</h2>
+        </div>
+      </section>
+      <section id="icons-secction-2">
+        <ul class="social-media-icons-2">
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/M.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/Paz.png" class="img-icons"></a>
+          </li>
+        </ul>
+      </section>
+      <div class="actions">
+        <button disabled class="Normalbc">Get my resume</button>
+      </div>
+    </div>
+    <div class="skill">
+      <ul class="skill-list">
+        <li>
+          <ul class="languages-menu">
+            <li class="languages">Languages<button disabled class="f-botton"><img src="img/v.png" alt="sub-menu"
+                  class="v-img"></button>
+            </li>
+            <li class="jmenu"><img src="img/JS (1).png" class="js-img">
+              <p class="jmenu-txt">JavaScript</p>
+            </li>
+            <li class="jmenu"><img src="img/HTML.png" class="js-img">
+              <p class="jmenu-txt">HTML</p>
+            </li>
+            <li class="jmenu"><img src="img/CSS.png" class="js-img">
+              <p class="jmenu-txt">CSS</p>
+            </li>
+          </ul>
```

Comment:
- Kindly make sure you dispose correctly your element  and use the appropriate font as is shown in the figma design it will make your page looks more beautiful :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/227680188-61a5a396-a602-4a84-af36-241e525ad8cf.png)



## Figma design 

![image](https://user-images.githubusercontent.com/95297251/227680115-343a5954-c5f3-4c1a-a4f0-b2e10d7edfc4.png)